packages/devtools_app/test/vm_developer/object_inspector/object_viewport_test.dart

```diff
@@ -44,6 +49,8 @@ void main() {
 
     testObjectInspectorViewController = TestObjectInspectorViewController();
 
+    // MockClassObject setUp
```

Comment:
Maybe remove the space between this comment and where you're assigning `mockClassObject`. I originally thought this was code you had left commented out, so this comment might be worth making more descriptive or may not be worth keeping.